tests/Doctrine/Tests/Common/DataFixtures/Purger/MongoDBPurgerTest.php

```diff
@@ -54,17 +52,28 @@ public function testPurgeKeepsIndices()
         $collection = $dm->getDocumentCollection(self::TEST_DOCUMENT_ROLE);
         $collection->drop();
 
-        $this->assertCount(0, $collection->getIndexInfo());
+        $this->assertIndexCount(0, $collection);
 
         $role = new Role();
         $role->setName('role');
         $dm->persist($role);
         $dm->flush();
 
-        $schema = $dm->getSchemaManager()->ensureDocumentIndexes(self::TEST_DOCUMENT_ROLE);
-        $this->assertCount(2, $collection->getIndexInfo());
+        $dm->getSchemaManager()->ensureDocumentIndexes(self::TEST_DOCUMENT_ROLE);
+        $this->assertIndexCount(2, $collection);
 
         $purger->purge();
-        $this->assertCount(2, $collection->getIndexInfo());
+        $this->assertIndexCount(2, $collection);
+    }
+
+    private function assertIndexCount(int $expectedCount, $collection)
```

Comment:
Please add a return type declaration (this repo uses php 7.2), and a phpdoc comment for `$collection` so that we know what types are supported.